README.md

```diff
@@ -1,264 +1,505 @@
+# NAMETBD: A design system integrating to Pattern Lab and a Drupal 8 theme
+
 [![Build Status](https://travis-ci.org/phase2/pattern-lab-starter.svg?branch=master)](https://travis-ci.org/phase2/pattern-lab-starter)
 
-# Prerequisites
+NAMETBD is an opinionated set of tools and examples to:
+
+1. Build an application-agnostic **design system**
+1. Apply that design system to a locally-served **Pattern Lab** for rapid prototyping
+1. Apply that design system to a **Drupal theme**
+
+In depth documentation about frontend approach using this project at [Phase2 Frontend Docs](https://phase2.github.io/frontend-docs/)
+
+## Prerequisites
+
+- [Node `^8.0.0`](https://nodejs.org)
+- [PHP `5.6`, `^7.0.0`](https://php.net)
+- [Composer `^1.0.0`](https://getcomposer.org)
 
-- [Node](https://nodejs.org) v6, v8 is required for pa11y testing
-- PHP 5.4, 5.5, 5.6, OR 7 (5.3 might work, no promises)
-- [`composer`](https://getcomposer.org)
+## Provides
 
-# QuickStart
+- Drupal theme and Pattern Lab app
+- Strict [Atomic Design](http://atomicdesign.bradfrost.com/) component structure
+- Webpack bundling of all CSS, javascript, font, and static image assets for multiple targets (Drupal theme and Pattern Lab)
+- Webpack Dev Server for local hosting and auto asset injection into Pattern Lab and Drupal
+- Auto namespace addition into Drupal theme and Pattern Lab. Within any twig file, `@atoms/thing.twig` means the same thing to Drupal theme and Pattern Lab.
```

Comment:
Since I am just starting on this it is possible this is linked elsewhere this document, but perhaps a link to an explanation on Twig namespace paths for people unfamiliar (such as http://symfony.com/doc/current/templating/namespaced_paths.html)

Reply:
Added!